core/src/main/scala/sttp/tapir/EndpointIO.scala

```diff
@@ -472,25 +474,27 @@ object EndpointIO {
     def of[T](t: T, name: Option[String] = None, summary: Option[String] = None): Example[T] = Example(t, name, summary)
   }
 
-  case class Info[T](description: Option[String], examples: List[Example[T]], deprecated: Boolean) {
+  case class Info[T](description: Option[String], examples: List[Example[T]], deprecated: Boolean, extensions: Vector[Extension[_]]) {
```

Comment:
I'm wondering if maybe we need a better name here. `Extensions` looks very generic - it might not be that obvious that these are documentation extensions. Maybe simply `DocumentationExtension`? Or `DocsExtension`? Sth that would point to the fact, that these extensions ar epurely documentational.

Reply:
I've changed the name everywhere except for methods in `Endpoint.scala` and `EndpointIO.scala`, do you think we should change their names as well? In this case adding extensionis will look like this: `.docsExtension(...)`. A bit more messy but more clear. What do you think?

Reply:
I think I'd rename the method as well if you don't mind. Who knows, we might have some non-doc extensions in the future ;)

Reply:
Sure, done